core/src/test/scala/integration/kafka/api/IntegrationTestHarness.scala

```diff
@@ -84,14 +84,6 @@ abstract class IntegrationTestHarness extends KafkaServerTestHarness {
   private def insertControllerListenersIfNeeded(props: Seq[Properties]): Unit = {
     if (isKRaftTest()) {
       props.foreach { config =>
-        // Add the CONTROLLER listener to "listeners" if it is not already there.
```

Comment:
Do we still need this method if we are no longer inserting controller listeners? Maybe we could rename it if the security config below is still needed?

Reply:
You are right. I removed this method. `QuorumTestHarness` only really supports `PLAINTEXT` for the controller listener. I think we are going to need additional changes to `QuorumTestHarness` to support some of the other security protocols.